### ISSUE TITLE
HEPL!! Problems in the work of the plug-in with the player version v7.0.3

### ISSUE THREAD

**Issue by inerc** [videojs/videojs-contrib-quality-levels#40]
Hi, I updated the player version. And I had a mistake.
```javascript
constructor(player: Player, options: Options) {
    super(player, options)
    this.options = options
    const qualityLevels = player.qualityLevels()
    this.quality = []
    qualityLevels.on('addqualitylevel', event => {
      this.quality.push(event.qualityLevel)
      this.update()
    })
  }```

Error:
`player.qualityLevels() undefined`
I can not find the cause of the problem. help me please

**Comment by gkatsev** [comment]
We have a pending PR to bring vjs 7 support to npm/bundler users: #39.

**Comment by inerc** [comment]
Thank you, guys. We are very much waiting for fix)

**Comment by gkatsev** [comment]
Should be fixed via in 2.0.5.
